scalableminds/webknossos
# Fetching only datasets according to filter criteria

### URL of deployed dev instance (used for testing):
- https://onlyfetchneededdatasets.webknossos.xyz

### Steps to test:
- Open the dashboard and the network tab of your browser
- If you change the filter criteria of datasets to "Only show available datasets" the request should have the parameter `?isActive=true`
- If you select the option "Only show missing datasets" the parameter should be `?isActive=false`
- If you select the option "Show all datasets" the request shouldn't contain any parameters

### Issues:
- fixes #4137

------
